app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -370,10 +368,11 @@ public void showBadImagePopup(Integer errorCode) {
             DialogUtil.showAlertDialog(getActivity(),
                     getString(R.string.warning),
                     errorMessageForResult,
-                    () -> deleteThisPicture(),
-                    () -> {
-                        uploadItem.setImageQuality(ImageUtils.IMAGE_KEEP);
+                    () -> { uploadItem.setImageQuality(ImageUtils.IMAGE_KEEP);
```

Comment:
`uploadItem.setImageQuality(ImageUtils.IMAGE_KEEP);` should be on a separate line to the `{` unless it is a `statement lambda` ie one method call or value

For example `deleteThisPicture();` below can have the braces and colon removed to become a statement lambda.

It can also go further and become a method reference `::deleteThisPicture` because the input parameters `()` are the same as the parameter for the method `deleteThisPicture()`